Workspace/src/ik/src/Roboarm.cpp

```diff
@@ -1,13 +1,18 @@
 #include <cmath>
 #include <iostream>
 
-#include "../include/ik/Roboarm.h"
+#include "Roboarm.h"
+#include <Eigen/Core>
```

Comment:
These should be in the header

Reply:
I disagree, headers should only conclude what is necessary, otherwise every package linking to ik will need to add Eigen as a dependency